solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1619,6 +1691,9 @@ impl pallet_registrar::Config for Runtime {
     type DepositAmount = DepositAmount;
     type RegistrarHooks = StarlightRegistrarHooks;
     type RuntimeHoldReason = RuntimeHoldReason;
+    // TODO: replace TestWeightInfo when we use proper weights on paras_registrar
+    type InnerRegistrar =
+        InnerStarlightRegistrar<AccountId, Registrar, paras_registrar::TestWeightInfo>;
     type WeightInfo = pallet_registrar::weights::SubstrateWeight<Runtime>;
 }
 
```

Comment:
Can we create an issue to prevent using the relay registrar with dispatch filters? does not have to be done in this PR

Reply:
Sure, issue created https://opslayer.atlassian.net/browse/MD-517?atlOrigin=eyJpIjoiYjlmYjkxMmZlNzI4NDJmZmFmMDkwM2UxMTdkZDliYTUiLCJwIjoiaiJ9

Reply:
_resolved_